zsh/.zsh.d/.functions

```diff
@@ -124,9 +124,9 @@ pacu() {
             sudo pacman -Sc --noconfirm
             sudo pacman-optimize
         fi
-        if [ -x "$(command -v yaourt)" ]; then
-            printf '\e[1mUpdating yaourt packages\e[0m\n'
-            yaourt -Syua --noconfirm
+        if [ -x "$(command -v yay)" ]; then
+            printf '\e[1mUpdating yay packages\e[0m\n'
```

Comment:
They seem to call the tool `Yay` instead of `yay` on https://github.com/Jguer/yay. Can we change that?

Reply:
In the command line, the tool is in lowercase. But in the print, we can change it to: "Updating Yay packages". What do you think?

Reply:
Yes, that's also what I was thinking.

Reply:
I have updated the 'Y' in printf